README.md

```diff
@@ -220,6 +223,162 @@ For example, add the following to the template `index.html`, inside the `entry-c
 {% endif %}
 ```
 
+## The new optional templates files for inline galleries, images and lightboxes
+
+There are three corresponding template files.
+
+### Inline gallery
+
+When the variable `PHOTO_INLINE_GALLERY_TEMPLATE` is set to true,
+a gallery could be placed everywhere in an article.
+An inline gallery is introduced in Markdown or in reStructuredText as:
+
+```
+  gallery::{photo}mygallery
+```
+
+The template file to render inline galleries is loaded.
+The name of this file is by default ``"inline_gallery.html"```
+and could be changed with the  `PHOTO_INLINE_GALLERY_TEMPLATE`  variable.
+The substituted variable is `gallery` which is the previous tuple with five elements.
+An example of this file is:
+```html
+<div class="gallery">
+    {% for title, gallery in galleries%}
+        <h1>{{ title }}</h1>
+            {% for name, photo, thumb, exif, caption in gallery %}
+                    <a href="{{ SITEURL }}/{{ photo }}" title="{{ name }}" exif="{{ exif }}" caption="{{ caption }}"><img src="{{ SITEURL }}/{{ thumb }}"></a>
+            {% endfor %}
+    {% endfor %}
+</div>
+```
+
+The `PHOTO_INLINE_GALLERY_PATTERN`
+variable in the ``pelicanconf.py`` file defined the pattern to look for both the
+```gallery_name``` and the ``options``, used to transmit options
+to the ``inline_gallery.html`.
+Its default value is
+
+```python
+r"gallery(\[(?P<options>[\w,=]+)\])?::(?P<gallery_name>[/{}\w_-]+)"
+```
+An inline gallery transmitting an option
+is introduced in Markdown or in reStructuredText as:
+
+```
+  gallery[reverse]::{photo}mygallery
+```
+
+The ``"inline_gallery.html"``` template file is then modified
+accordingly to take into accound this option:
+
+```html
+<div class="gallery">
+    {% if options and options['reverse'] %}
+      {% set use_reverse = True %}
+    {% else %}
+      {% set use_reverse = False %}
+    {% endif %}
+    {% for title, gallery in galleries%}
+        <h1>{{ title }}</h1>
+            {% for name, photo, thumb, exif, caption in (gallery | sort(reverse=use_reverse,attribute="filename")) %}
+                    <a href="{{ SITEURL }}/{{ photo }}" title="{{ name }}" exif="{{ exif }}" caption="{{ caption }}"><img src="{{ SITEURL }}/{{ thumb }}"></a>
+            {% endfor %}
+    {% endfor %}
+</div>
+```
+
+The ``options`` variable is a python ``dict``:
+it it is defined as a coma-separated list of ``keys``
+and ``value`` pairs, e.g.
+
+```
+  gallery[reverse,width=20em]::{photo}mygallery
+```
+
+When there is no explicit value, the value is a ``bool``, set to ``True``.
+When the value is explicitely ``False``, it is also converted to ``bool``,
+otherwise it is left as a ``string``.
+
```

Comment:
It would be helpful to list the context variables available to the template here, the way you've done for the image template, below.